pkg/plugin/instance/libvirt/instance.go

```diff
@@ -31,9 +38,10 @@ type infrakitMetadata struct {
 	// <infrakit xmlns="https://github.com/docker/infrakit">...</infrakit>
 	// See https://github.com/golang/go/issues/13400,
 	// https://github.com/golang/go/issues/9519 and various linked issues.
-	XMLName   xml.Name              `xml:"https://github.com/docker/infrakit infrakit"`
-	LogicalID string                `xml:"logicalid"`
-	Tags      []infrakitMetadataTag `xml:"tag"`
+	XMLName          xml.Name                  `xml:"https://github.com/docker/infrakit infrakit"`
+	LogicalID        string                    `xml:"logicalid"`
+	Tags             []infrakitMetadataTag     `xml:"tag"`
+	MetadataDiskInfo *infrakitMetadataDiskInfo `xml:"metadata-disk",omitempty`
```

Comment:
I think the syntax is `xml:"metadata-disk,omitempty"`  This was giving an go vet error.